### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+**ACP Extension version used:**
+
+*version goes here (list all the extension versions if using multiple)*
+
+- iOS: Find these in the Podfile.lock file generated for your application
+- Android: Run the following command with gradle: *./gradlew :app:dependencyInsight* and extract the com.adobe.marketing.mobile extension versions
+
+**Platform:**
```
Comment (nporter-adbe) [inline .github/ISSUE_TEMPLATE/issue_template.md:28]: We should ask for the platform and the version. E.g. iOS 10.1, and maybe Xcode/Android studio version.

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<!--- STOP! Before you open an issue please search this repository's issues to see if it has already been reported. This helps reduce duplicate issues from being created. -->
+
+**Short description of the issue**:
+
+*description here*
```
Comment (sbenedicadb) [inline .github/ISSUE_TEMPLATE/issue_template.md:5]: The filler text between our headers should be comments, so there's no need to delete them

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+**Installation method**:
+
+- [ ] Dependency manager (CocoaPods/Maven)
+- [ ] Manual linking
+
+**How easy is to reproduce?**
+
+- [ ] Easy, able to reproduce 100% of the times
+- [ ] Intermmitent (provide percentage)
```
Comment (sbenedicadb) [inline .github/ISSUE_TEMPLATE/issue_template.md:41]: `intermittent`

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- [ ] Not able to reproduce, but have crash report
+
+**Can you share SDK logs or crash reports?**
+
+*SDK logs or crash reports can help us identify the issue much faster. Please note that this is a public repository so you might want to obfuscate sensitive information.*
+
+**IDE and version used (optional)**:
```
Comment (sbenedicadb) [inline .github/ISSUE_TEMPLATE/issue_template.md:48]: should add `Other (specify)` as an option

**Pull request merged**